source/atlas-cli-changelog.txt

```diff
@@ -14,6 +14,21 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.28.0:
+
+{+atlas-cli+} 1.27.1
+--------------------
+
+Released 19 September 2024
+
+- Adds the ``--file`` flag to the
+  :ref:`atlas-organizations-invitations-invite` and
+  :ref:`atlas-organizations-invitations-update` commands.
+- Adds support for MongoDB 8.0 for local deployments when you run
+  :ref:`atlas-deployments-setup`.
+- Deprecates the ``atlas-projects-invitations`` commands. Use the 
+  :ref:`atlas-organizations-invitations` commands for invitations.
```

Comment:
The download logs usage info also improved, but I'm not sure if it's worth an entry.
https://github.com/mongodb/mongodb-atlas-cli/pull/3253